elastica/contact_forces.py

```diff
@@ -423,6 +425,102 @@ def _calculate_contact_forces_self_rod(
                 external_forces_rod[..., j + 1] += net_contact_force
 
 
+@numba.njit(cache=True)
+def _calculate_contact_forces_rod_mesh_surface(
+    faces_normals,
+    faces_centers,
+    element_position,
+    position_idx_array,
+    face_idx_array,
+    surface_tol,
+    k,
+    nu,
+    radius,
+    mass,
+    position_collection,
+    velocity_collection,
+    internal_forces,
+    external_forces,
+    lengths,
+):
+    """
+    This function computes the plane force response on the element, in the
```

Comment:
Can we update docs?

Reply:
Could you elaborate on what should we update?